adapter/internal/discovery/xds/server.go

```diff
@@ -251,7 +251,7 @@ func UpdateAPI(apiContent config.APIContent) {
 
 	validationErr := mgwSwagger.Validate()
 	if validationErr != nil {
-		logger.LoggerOasparser.Errorf("Validation failed for the API. %s:%s", mgwSwagger.GetTitle(), mgwSwagger.GetVersion())
+		logger.LoggerOasparser.Errorf("Validation failed for the API %s:%s of Organization %s", mgwSwagger.GetTitle(), mgwSwagger.GetVersion(), organizationID)
```

Comment:
Is ot possible to print the org name as well apart from ID

Reply:
The organization id is coming in the api.yaml is or the event is printed here.